declarations_site/catalog/management/commands/loadcatalog.py

```diff
@@ -15,13 +16,35 @@
 
 class Command(BaseCommand):
     args = '<file_path>'
-    help = 'Loads the catalog of declaration into the persistence storage'
+    help = ('Loads the CSV catalog of existing declarations '
+            'into the persistence storage')
```

Comment:
Somehow I was hoping people now have these higher resolution displays and don't need 80 chars limits any more :)